examples/IB/explicit/ex7/example.cpp

```diff
@@ -0,0 +1,515 @@
+       const int n_proc = IBTK_MPI::getNodes();
+       int rank = IBTK_MPI::getRank();
+
+
+        // Create Text File which Stores the locations of each IB point.
+        std::ofstream fout;
+
+        if(rank == 0){
+            fout.open("Coordinates.txt",std::ios::out);
+        }
+
+        //Vector to store the number of nodes on each processor
+        std::vector<int> nodes_per_proc(n_proc);
+
+
+
+
+        // Main time step loop.
+        double loop_time_end = time_integrator->getEndTime();
+        double dt = 0.0;
+        while (!MathUtilities<double>::equalEps(loop_time, loop_time_end) && time_integrator->stepsRemaining())
+        {
+            iteration_num = time_integrator->getIntegratorStep();
+            loop_time = time_integrator->getIntegratorTime();
+
+            pout << "\n";
+            pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
+            pout << "At beginning of timestep # " << iteration_num << "\n";
+            pout << "Simulation time is " << loop_time << "\n";
+
+
+            update_target_points(patch_hierarchy, ib_method_ops->getLDataManager(), loop_time, dt);
+
+
+            dt = time_integrator->getMaximumTimeStepSize();
+            time_integrator->advanceHierarchy(dt);
+            loop_time += dt;
+
+            pout << "\n";
+            pout << "At end       of timestep # " << iteration_num << "\n";
+            pout << "Simulation time is " << loop_time << "\n";
+            pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
+            pout << "\n";
+
+            //Print the Eulerian Coordinates of the IB points which constitute the Lagrangian structure.
+            const int finest_ln = patch_hierarchy->getFinestLevelNumber();
+            Pointer<LData> X_data = ib_method_ops->getLDataManager()->getLData("X", finest_ln);
+            Vec X_vec = X_data->getVec();
+            double* x_vals;
+            int ierr = VecGetArray(X_vec, &x_vals);
+            IBTK_CHKERRQ(ierr);
+            Pointer<LMesh> l_mesh = ib_method_ops->getLDataManager()->getLMesh(finest_ln);
+            const std::vector<LNode*>& local_nodes = l_mesh->getLocalNodes();
+            int num_nodes = local_nodes.size();
+            //Send to processor zero the number of IB points on each processor
+            if(rank == 0)
+            {
+                nodes_per_proc[0] = num_nodes;
+            }
+
+            if(rank !=0)
+            {
+                MPI_Send(&num_nodes,1,MPI_INT,0,0,IBTK_MPI::getCommunicator());
+            }
+            else
+            {
+                for(int r = 1; r<n_proc; ++r)
+                {
+                    MPI_Recv(&nodes_per_proc[r],1,MPI_INT,r,0,IBTK_MPI::getCommunicator(),MPI_STATUS_IGNORE);
+                }
+            }
+
+            IBTK_MPI::barrier();
+            //initialize vector to store the nodal coordinates on each processor
+            std::vector<double> Local_Coords(num_nodes*NDIM);
```

Comment:
```suggestion
            std::vector<double> local_coordinates(num_nodes*NDIM);
```
we typically don't use capital letters for variables.